docs/standalone/building-on-electrumsv/node-wallet-api.rst

```diff
@@ -756,6 +756,13 @@ call processing are described above.
       :Message: | ``JSON value is not a null as expected``
                 | This is an intentional incompatibility as we do not support this parameter.
 
+    - :Code: -8 ``RPC_INVALID_PARAMETER``
```

Comment:
Thanks for adding this. Please put it in order (the errors are in order of the numerical code if you look).